data/participants.json

```diff
@@ -1038,6 +1038,16 @@
    "github" : "https://github.com/mivishvajeet",
    "twitter" : "https://twitter.com/mivishvajeet",
    "referral" : ""
+},
+{
+			"name": "Rajat",
+			"imageurl": "Opencode-Collaborative-19/assets/img/participants/RajatGithubProfile.jpg
+      ",
```

Comment:
fix this part, there is extra space because of which your json is linted properly